src/templates/common/_rss/head.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" media="screen" href="/rss.xsl"?>
```

Comment:
> `href="/rss.xsl"`

I'm not sure if this will actually work, let me know if this should be replaced by something else.